### PR TITLE
Assert container parameter types

### PR DESCRIPTION
**Pull request by greg0ire**
Not sure why PHPStan did not spot this before, but now it does.

### CODE REVIEW

File: Command/DoctrineCommand.php [hunk 1]
```diff
@@ -137,4 +156,12 @@ private static function injectContainerToMigrations(ContainerInterface $containe
             $migration->setContainer($container);
         }
     }
+
+    public static function getStringParameter(ContainerInterface $container, string $name): string
```
Comment (andrew-demb) [inline Command/DoctrineCommand.php]:
```suggestion
    private static function getStringParameter(ContainerInterface $container, string $name): string
```

Comment (greg0ire) [reply Command/DoctrineCommand.php]: :thinking: why would I do that?

Comment (andrew-demb) [reply Command/DoctrineCommand.php]: Because this is not used anywhere except this class.

Comment (greg0ire) [reply Command/DoctrineCommand.php]: Oh sorry I read that backwards, fully agree!

**Review by SenseException — APPROVED** [review]

**Pull request merged**